### PR TITLE
Jira V2 - fixed command jira-get-id-by-attribute to return only active users

### PR DESCRIPTION
**Pull request by abaumgarten**
## Status
- [] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/34485
fixes: https://github.com/demisto/etc/issues/34676
related: https://github.com/demisto/dockerfiles/pull/4434

## Description
- Fixed an issue where the command **jira-get-id-by-attribute** returned a list of active and inactive users. Now, the new endpoint returns a list of only active users.
The customer's issue probably caused by a long list of inactive users and then the specific active user was not found. 
Updated TBP to use UserName instead of Email address because when using Auth1 email address returns no results.

the new endpoint docs: 
https://developer.atlassian.com/cloud/jira/platform/rest/v3/api-group-user-search/#api-rest-api-3-user-picker-get

## Minimum version of Demisto
- [x] 5.0.0

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/Jira/Integrations/JiraV2/JiraV2.py [hunk 1]
```diff
@@ -197,14 +199,14 @@ def search_user(query: str, max_results: str = '50'):
...
-    url = f"rest/api/latest/users/search?query={query}&maxResults={max_results}"
+    url = f"rest/api/latest/user/search?query={query}&maxResults={max_results}"
     res = jira_req('GET', url, resp_type='json')
     return res
 
 
 def get_account_id_from_attribute(attribute: str, max_results: str = '50') -> Union[CommandResults, str]:
     """
-    https://developer.atlassian.com/cloud/jira/platform/rest/v3/api-group-user-search/#api-rest-api-3-user-search-get
+    https://developer.atlassian.com/cloud/jira/platform/rest/v3/api-group-user-search/#api-rest-api-3-user-picker-get
```
Comment (Bargenish) [inline Packs/Jira/Integrations/JiraV2/JiraV2.py:209]: no need to change. you're still using the user/search endpoint and not the picker one.

**Review by Bargenish — APPROVED** [review]

File: Packs/Jira/ReleaseNotes/1_2_13.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Atlassian Jira v2
+- Fixed an issue where the command **jira-get-id-by-attribute** returned a list of active and inactive users. Now, the new endpoint returns a list of only active users.
```
Comment (amshamah419) [inline Packs/Jira/ReleaseNotes/1_2_13.md:4]:
```suggestion
- Fixed an issue where the **jira-get-id-by-attribute** command would return a list of active and inactive users. Now, the new endpoint returns only a list of active users.
```

**Review by yuvalbenshalom — CHANGES_REQUESTED** [review]
@abaumgarten see my comment here https://github.com/demisto/etc/issues/34485#issuecomment-792263955

I do not understand what is the difference between the endpoints

**Comment by abaumgarten** [comment]
> @abaumgarten see my comment here [demisto/etc#34485 (comment)](https://github.com/demisto/etc/issues/34485#issuecomment-792263955)
> 
> I do not understand what is the difference between the endpoints

The old one:
image

The new one:
image

Please see my comment in the issue

**Review by yuvalbenshalom — APPROVED** [review]

**Pull request merged**